virttest/env_process.py

```diff
@@ -911,6 +911,8 @@ def postprocess(test, params, env):
         guest_dmesg_log_file = utils_misc.get_path(test.debugdir, guest_dmesg_log_file)
         living_vms = [vm for vm in env.get_all_vms() if vm.is_alive()]
         for vm in living_vms:
+            if vm.is_paused():
+                vm.resume()
```

Comment:
Ideally test case should restore the `vm` states after the test is done, we expect the VM in running state, even if not running, `waitforlogin` method will error out.
My opinion not to handle here, otherwise we might have to handle the corner cases of other states like `in shutdown` etc, 
Can you point me the failures log?

Reply:
@hsmj1412 Just gone through the logs, we need further refining of `only running` vms, to be used,
it is good  not to change the vm states after test is over.

Reply:
@hsmj1412 tried addressing this way https://github.com/avocado-framework/avocado-vt/pull/1181, can you try and let me know your results and comments, Thanks in advance.